src/Agent.php

```diff
@@ -54,14 +54,11 @@ public function asVersion($version) {
             if (! empty($versioned_acct)) {
                 $result['account'] = $versioned_acct;
             }
-        }
-        elseif (isset($this->mbox_sha1sum)) {
+        } else if (isset($this->mbox_sha1sum)) {
```

Comment:
For some reason phpcc won't consider these lines covered when written like this:

```php
elseif (condition) {
}
elseif (condition) {
}
```

This might just be me but I don't think so.